README.md

```diff
@@ -8,6 +8,7 @@ npm test -> No implemeted yet
 npm run start -> it will start the server using pm2
 npm run stop  -> will stop the server
 npm run build -> it will compile .ts files into .js for node server usage
+npm run populate-db -> run the script to populate db with initial values. Before run this task make sure you build the project first.
```

Comment:
The readme should include more information about running the application. It would be nice to have a step by step guide

Reply:
I'll take it into account for the next feature.